lib/net.js

```diff
@@ -43,9 +43,10 @@ const async_id_symbol = process.binding('async_wrap').async_id_symbol;
 const { newUid, setInitTriggerId } = require('async_hooks');
 const nextTick = require('internal/process/next_tick').nextTick;
 const errors = require('internal/errors');
+const dns = require('dns');
 
-var cluster;
-var dns;
+// Lazily loaded
+var cluster = null;
```

Comment:
This could probably be required up front as well but `cluster` is likely not required in many places that use `net` and therefore I kept it as is.